framework/managed-context/pom.xml

```diff
@@ -29,9 +29,9 @@
     </parent>
     <dependencies>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>${cs.log4j.version}</version>
```

Comment:
do we (still) need the version here?

Reply:
Not sure, did n't want to refactor/experiment; so I've left all build config as is just replaced log4j with reload4j.